bot/code_review_bot/tasks/lint.py

```diff
@@ -110,14 +110,32 @@ class MozLintTask(AnalysisTask):
     valid_states = ("failed",)
     skipped_states = ("completed",)
 
+    @property
+    def linter(self):
+        # Detect linter from task name
+        if not self.name.startswith("source-test-mozlint-"):
+            return
+        return self.name[20:]
+
+    @property
+    def display_name(self):
+        if self.linter:
+            return f"{self.linter} (by MozLint)"
```

Comment:
```suggestion
            return f"{self.linter} (by Mozlint)"
```
https://firefox-source-docs.mozilla.org/code-quality/lint/index.html